cmd/pack3d/main.go

```diff
@@ -30,12 +33,20 @@ func timed(name string) func() {
 }
 
 func main() {
+
+	type Data struct {
+		Filename         string
+		Transformation   [4][4]float64
+	}
+
 	var (
 		singleStlSize []fauxgl.Vector
 	    done          func()
 	    totalVolume   float64
 		dimension     []float64
 		ntime         int
+		filenames     []string
```

Comment:
filename of what? source stl?  maybe 'sources_stls' or 'packing_stls' is a more clear name.